opt_einsum/contract.py

```diff
@@ -31,15 +32,21 @@ class PathInfo(object):
         produced during the contraction.
     """
 
-    def __init__(self, contraction_list, input_subscripts, output_subscript,
-                 indices, scale_list, naive_cost, opt_cost, size_list):
+    def __init__(self, contraction_list, input_subscripts, output_subscript, indices,
+                 path, scale_list, naive_cost, opt_cost, size_list, size_dict):
         self.contraction_list = contraction_list
         self.input_subscripts = input_subscripts
         self.output_subscript = output_subscript
+        self.path = path
         self.indices = indices
         self.scale_list = scale_list
         self.naive_cost = naive_cost
```

Comment:
Maybe save `opt_cost` and `naive_cost` as Decimal and then add the speedup here as we seem to access it in the docs a bit? Should be fine to import decimal overall as it is quite lightweight.

Reply:
Yes, will do.